boards/posix/native_posix/tracing.c

```diff
@@ -12,6 +12,7 @@
 #include <stdio.h>  /* for printfs */
 #include <stdarg.h> /* for va args */
 #include <unistd.h>
+#include <kernel.h>
```

Comment:
why add this?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/20119#discussion_r341810118